languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,76 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PAS_DE_VIRGULE" name="pas de virgule" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">et|ou</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\2</suggestion>
+              <example correction="et">Elle a acheté des radis<marker>, et</marker> des choux.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token>c'</token>
+                  <token skip="-1">est</token>
+                  <marker>
+                      <token>,</token>
+                      <token regexp="yes">qui|que</token>
+                  </marker>
+              </pattern>
+              <message>La virgule doit être omise dans ce contexte.</message>
+              <suggestion>\4</suggestion>
+              <example correction="qui">C'est lui<marker>, qui</marker> part demain.</example>
+              <example correction="que">C’est l’eau laissée dans la piscine<marker>, que</marker> j'ai vidée.</example>
+          </rule>
```

Comment:
We might want to look out for "incisives":

C'est lui, que tout le monde avait oublié, qui sauva la journée.

I think this can be prevented from matching by writing an AP for `, qu[ei]` followed (skip="-1") by another `, qu[ei]`.